<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulateWindow</class>
 <widget class="QWidget" name="SimulateWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1701</width>
    <height>945</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="dataBar" native="true">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>520</y>
     <width>516</width>
     <height>230</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLabel" name="lableShowSimulateRate">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lable_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;方正粗黑宋简体&quot;;</string>
      </property>
      <property name="text">
       <string>当前模拟倍速：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="lableShowTime">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;方正粗黑宋简体&quot;;</string>
      </property>
      <property name="text">
       <string>模拟时长：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;方正粗黑宋简体&quot;;</string>
      </property>
      <property name="text">
       <string>通过车辆数量：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="lableShowPassCar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="dataBar_2" native="true">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>630</y>
     <width>516</width>
     <height>304</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="left1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;方正粗黑宋简体&quot;;</string>
      </property>
      <property name="text">
       <string>从左往右拥堵程度：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="lableJamLeftToRight">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="left2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;方正粗黑宋简体&quot;;</string>
      </property>
      <property name="text">
       <string>从右往左拥堵程度：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="lableJamRightToLeft">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="left3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;方正粗黑宋简体&quot;;</string>
      </property>
      <property name="text">
       <string>左转拥堵程度：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="lableJamTurnLeft">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="left4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;方正粗黑宋简体&quot;;</string>
      </property>
      <property name="text">
       <string>右转拥堵程度：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="lableJamTurnRight">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>380</y>
     <width>511</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget
{
border-image: url(:/res/road1.png);
}</string>
   </property>
   <widget class="QWidget" name="colorJamToRight" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="colorJamToLeft" native="true">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="colorJamTurnRight" native="true">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>31</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="colorJamTurnLeft" native="true">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>31</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
